src/lib/helpers.ts

```diff
@@ -220,6 +220,53 @@ export function getOvalConfigs(input: string): OvalConfigs {
   throw new Error(`Value "${input}" is valid JSON but is not OvalConfigs records`);
 }
 
+// Get OvalHeaderConfigs from the header string or throw an error if the header is invalid.
+export const getOvalHeaderConfigs = (
+  header: string | string[] | undefined,
+  ovalConfigs: OvalConfigs,
+): { valid: boolean; ovalHeaderConfigs: OvalHeaderConfigs | undefined } => {
+  if (!header) return { valid: true, ovalHeaderConfigs: undefined };
+  if (typeof header !== "string") return { valid: false, ovalHeaderConfigs: undefined };
+
+  let ovalHeaderConfigs: OvalHeaderConfigs;
+  let valid = true;
+  try {
+    ovalHeaderConfigs = JSON.parse(header);
```

Comment:
OOC, why did you decide to make this an object with a single field rather than just an array of addresses?

Basically, why this?
`{ unlockAddresses: ["0x1234", "0x4567"] }`

Rather than this?
`["0x1234", "0x4567"]`

Note: if we did the latter, we might want to change the header name to something like "x-oval-instances" or "x-oval-addresses"

Reply:
Yes, I agree. Initially, I was thinking of eventually having more configurations passed in that object, but I think it's better to make it simpler.